### ISSUE TITLE
Sort buttons don't have enough contrast

### ISSUE THREAD

**Issue by plweil** [xaksis/vue-good-table#899]
- [x] Bug

The sort buttons for the Black-Rhino and Nocturnal themes have too little contrast, which means they fail [WCAG 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

I am addressing these in a forthcoming PR, which also addresses #888 and #851. At this point I am considering two options:

1. change the color of the buttons only

![image](https://user-images.githubusercontent.com/1033771/139848047-e503c113-bf71-4444-aecb-905d822c46d2.png)

2. add `focus` and `hover` colors that will visually highlight the entire `th` cell. This is a bit more complicated, but the result is an enhanced and even clearer visual indication that a column is sorted and how it is sorted.

![image](https://user-images.githubusercontent.com/1033771/139847494-bfbce2dd-2507-41fd-9df7-8d7be6a0014f.png)

Which of these is preferable?
